Source/WebCore/platform/graphics/mac/controls/SwitchThumbMac.mm

```diff
@@ -76,9 +76,12 @@
 
     auto inflatedTrackRect = inflatedRect(trackRect, size, outsets, style);
 
+    auto drawingThumbSide = inflatedTrackRect.height();
```

Comment:
I think you meant `size` rather than `side` here?

Reply:
I meant side as in each side has this length. A size would be a combination of a height and width in WebKit, I think.

Reply:
Ah, I think `drawingThumbLength` would be better then.